src/App.css

```diff
@@ -28,3 +28,131 @@ html {
 	justify-content: center;
 	font-family: 'Share Tech Mono', monospace;
 }
+
```

Comment:
EXCELLENT styling! Wow, so much work put into this! I'm curious why you chose to put the dialog box styling in the root (App) style folder? I created separate folders for most other components that connect to specific styling, and moving this could be an argument for consistency. Not a blocker, just a suggestion!

Reply:
Yeah, that makes sense, I will do this way!